maestro/schemas/workflow_schema.json

```diff
@@ -92,7 +92,22 @@
                       },
                       "template": {
                         "type": "string",
-                          "description": "template string for next prompt, {prompt} and {respomse} are replaced by the previos prompt and the user response"
+                        "description": "template string for next prompt, {prompt} and {respomse} are replaced by the previos prompt and the user response"
+                      }
+                    }
+                  },
+                  "loop": {
+                    "type": "object",
+                    "description": "looping construct",
+                    "items": {
+                      "type": "object",
+                      "properties": {
+                        "step": {
+                          "type": "string"
+                        },
+                        "until": {
+                          "type": "string"
+                        }
                       }
                     }
```

Comment:
The `items` is not necessary here.
so it should be
```yaml
                    "properties": {
                      "step": {
                        "type": "string"
                      },
                      "until": {
                        "type": "string"
                      }
                    }
```
instead of
```yaml
                    "items": {
                      "type": "object",
                      "properties": {
                        "step": {
                          "type": "string"
                        },
                        "until": {
                          "type": "string"
                        }
                      }
                    }
```

Reply:
OK fair. Good feedack. Addressing